src/common/utils/redis/helper.go

```diff
@@ -103,12 +107,126 @@ type Options struct {
 	maxRetry   int
 }
 
+var (
+	opt     *Options
+	optOnce sync.Once
+
+	defaultDelay    = int64(1) // 1 second
```

Comment:
make them configurable?

Reply:
configurations are implemented in `DefaultOptions` function